velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -696,6 +696,48 @@ TEST_F(DateTimeFunctionsTest, hour) {
   EXPECT_EQ(2, hour("1969-01-01 13:23:00.001"));
 }
 
+TEST_F(DateTimeFunctionsTest, minute) {
+  const auto minute = [&](const StringView timestampStr) {
+    const auto timeStamp =
+        std::make_optional(util::fromTimestampString(timestampStr));
+    return evaluateOnce<int32_t>("minute(c0)", timeStamp);
+  };
+
+  EXPECT_EQ(23, minute("2024-01-08 00:23:00.001"));
+  EXPECT_EQ(59, minute("2024-01-08 00:59:59.999"));
+  EXPECT_EQ(10, minute("2015-04-08 13:10:15"));
+  EXPECT_EQ(43, minute("1969-01-01 13:43:00.001"));
+
+  // Set time zone to Pacific/Apia (13 hours ahead of UTC).
+  setQueryTimeZone("Pacific/Apia");
+
+  EXPECT_EQ(23, minute("2024-01-08 00:23:00.001"));
```

Comment:
@acvictor, to see the effect of time zone setting, it may be better to test with half-hour timezone, e.g., "Indian/Cocos".

Reply:
Thanks @PHILO-HE! Added a test for UTC+5.5.